README.md

```diff
@@ -1,8 +1,9 @@
 # vaah-webdriverio
-Helpful classes to reduce code &amp; accelerate speed for writing test cases for WebDriver.io 
+Helpful classes to reduce code &amp; accelerate speed for writing test cases for WebDriver.IO 
 
 ## Objectives
 - Reduce code for `selecting` elements and verifying `asserts`
+- Code `re-usability` by writing the test scripts independent of the `data` and `POM`.
```

Comment:
@satyajeet-s001 write the full form of POM in brackets

Reply:
Updated sir!